packages/client/components/ReflectionCard/ReflectionCard.tsx

```diff
@@ -135,7 +137,9 @@ const ReflectionCard = (props: Props) => {
     }
   }
 
-  const readOnly = getReadOnly(reflection, phaseType as NewMeetingPhaseTypeEnum, stackCount)
+  const votePhaseReached = phases && isPhaseComplete(NewMeetingPhaseTypeEnum.group, phases)
```

Comment:
it looks like the only way this could be happening is `isEditing` is true.
If we merge this, that means `isEditing` could be true while `readOnly` is true, too, which scares me a little bit, but i don't think it could cause any major problems.

nit-- let's stick this in `getReadOnly` so all the readOnly logic lives in 1 place